modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/ApplicationAPI.macro

```diff
@@ -43,10 +43,18 @@ definition {
 				"apiApplicationToAPIEndpoints": [{
 					"httpMethod": "get",
 					"name": "${endpointName}",
-					"path": "${endpointPath}"
+					"path": "${endpointPath}",
+					"scope": {"name": "${scopeKey}"}
 				}]
 			''';
 
+			if (isSet(scopeKey)) {
```

Comment:
If we pass`scopeKey=company`, then the codes doesn't work correctly, please update here

Reply:
ok, I will update it soon